x-pack/plugins/monitoring/public/components/table/toolbar.js

```diff
@@ -13,6 +13,8 @@ import {
   KuiToolBarText
 } from '@kbn/ui-framework/components';
 
+import './toolbar.css';
+
```

Comment:
We've been keeping all our CSS separate from public/components, and we have a place in `public/less` that would be more suitable for the new `flex` rule in `public/less/components/table.less`